overall-equipment-effectiveness/workflows/Paper_Machine_Data_Simulator.yaml

```diff
@@ -0,0 +1,1969 @@
+resourceType: Flow
```

Comment:
You're only using the "Random Time" node coming off the "Headbox" case in the Switch Node. You could move it down to just above the Conditional coming off "Headbox".

<img width="744" alt="Screen Shot 2020-09-25 at 6 26 39 PM" src="https://user-images.githubusercontent.com/881769/94321173-e149b680-ff5c-11ea-805f-8d615ebbaae4.png">

Reply:
It is actually setting the time for every type of downtime event in the device: state node further down. I updated the note to eliminate confusion.